packages/field-base/test/input-control-mixin.test.js

```diff
@@ -369,6 +370,18 @@ const runTests = (defineHelper, baseMixin) => {
         element._preventInputDebouncer.flush();
         expect(element.hasAttribute('input-prevented')).to.be.false;
       });
+
+      it('should check the helper checkbox', async () => {
+        element.allowedCharPattern = '';
```

Comment:
Currently the test also passes without the fix. This needs to define a pattern, otherwise the check here does not pass because the pattern is empty:
https://github.com/vaadin/web-components/blob/e0cbce93b172afa042a6ead88d26dab8bd77aa67/packages/field-base/src/input-control-mixin.js#L191